app/Http/Kernel.php

```diff
@@ -22,6 +22,7 @@ class Kernel extends HttpKernel
         \App\Http\Middleware\TrimStrings::class,
         \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
         \App\Http\Middleware\QueryInfo::class,
+        \App\Http\Middleware\TransformsRequestKeys::class,
```

Comment:
Ja bym to jednak wolał jako opcjonalny middleware

jak będziemy mieć kod bazujący na obiektach domenowych (w każdym razie nie bezpośrednio na Modelach), to będziemy woleli, żeby pozostał camelCase

Reply:
Tez nie robienie w ogóle tej transformacji z automatu trochę bardziej narzuca "zastanów się nad każdym polem zanim przerzucisz je z requestu do modelu"

Reply:
takie rozwiązanie mamy np. sotsie, w sumie kiedy będzie sytuacja, że raz chcemy transformować na snake_case a raz nie? nie lepiej zachować jednolitość w całym systemie?

Reply:
może jak pojawi się taka potrzeba to wtedy zdejmiemy ten middleware i ustawimy opcjonalnie na konkretnych kontrolerach (ale wątpię, żeby zaszła taka potrzeba)

Reply:
no niech będzie